merge_reports.js

```diff
@@ -50,5 +66,5 @@ module.exports = (banditReport, gosecReport) => {
   } else {
     annotations = annotations.concat(gosecReport.annotations, banditReport.annotations)
   }
-  return { title, summary, annotations }
+  return { title, summary, annotations, text }
```

Comment:
I don't really like the idea of having `undefined` variables even though it works out in this case (the GitHub API will happily ignore it). Is there another way?

Reply:
The other two ways is to return in the if/else statements but that's something Eric told me long ago that he considers for not a good style.
Right @ericwb ? 

The other way is to check if the value is undefined in the end when you return which is little redundant. 

I used the current method on many occasions but I will change that just so we can see the alternative. I agree with you, you have a point.

Reply:
I suggest you initialize `text` as an empty string, that way it is functionally identical and we don't have to worry about possible undefined values.

Reply:
Yes, you are right.